### DISCUSSION TITLE
[suggestion] Exception-Ignore expression

### DISCUSSION THREAD

**Discussion by surfsky** [dotnet/csharplang#3495] [Language Ideas]
```
int n = ParseInt("21test") !! 0;
```

means
```
int n;
try
{
    n = ParseInt(21test");
}
catch
{
    n = 0;
}
```

**Comment by yaakov-h** [comment]
See the discussion on #220 about why ignoring exceptions is a bad idea.

**Comment by Niewiarowski** [comment]
This isn't Python or whatever other language you like to use. Stop trying to turn it into that. You've been spamming these issues like crazy. Every single issue you've created complains about stuff that is already pretty simple to do.

Take this issue for example. You could just do:
`int.TryParse("21test", out int n);`

It seems that you really don't know anything about the language and you're trying to force what you know from other languages into it. I think you should learn more about the language as it currently is before trying to make any suggestions. Thanks.

**Comment by jnm2** [comment]
@Niewiarowski Ouch. That sounds personal and not particularly welcoming. Please be aware of https://github.com/dotnet/csharplang/blob/master/CODE-OF-CONDUCT.md.
